integration-test/src/test/resources/trino-ci-testset/testsets/jdbc-mysql/00008_alter_catalog.txt

```diff
@@ -1,17 +1,17 @@
 CALL
 
-"gt_mysql_xxx1","jdbc-mysql","{""jdbc-url"":""jdbc:mysql://%/?useSSL=false"",""jdbc-user"":""trino"",""jdbc-password"":""ds123"",""jdbc-driver"":""com.mysql.cj.jdbc.Driver""}"
+"gt_mysql_xxx1","jdbc-mysql","{""jdbc-driver"":""com.mysql.cj.jdbc.Driver"",""jdbc-password"":""ds123"",""jdbc-url"":""jdbc:mysql://%/?useSSL=false"",""jdbc-user"":""trino""}"
 
```

Comment:
I changed the code to make the attributes ordered.

Reply:
No, make it order can avoid missmatch result in diffrent test environment

Reply:
Did you compare the string value during the test as you need to make it organized?

Reply:
No, I changed the serialization logic to make the properties key-value ordered.